src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/BFHelper.java

```diff
@@ -23,22 +23,14 @@ private BFHelper() {
     }
 
     public static void updateBfActionItems(CaseData caseData) {
-
         List<BFActionTypeItem> bfActions = caseData.getBfActions();
-
         if (bfActions != null && !bfActions.isEmpty()) {
-
             for (BFActionTypeItem bfActionTypeItem : bfActions) {
                 var bfActionType = bfActionTypeItem.getValue();
-
                 if (isNullOrEmpty(bfActionType.getDateEntered())) {
                     bfActionType.setDateEntered(UtilHelper.formatCurrentDate2(LocalDate.now()));
                 }
-                var dateEntered = bfActions.get(0).getValue().getDateEntered().substring(0, 10);
```

Comment:
It's pretty amazing that this functionality can be removed and yet it doesn't break a unit test.